cli/src/index.ts

```diff
@@ -42,8 +42,10 @@ type Args = {
   logLevel?: LogLevel;
   toolName?: string;
   toolArg?: Record<string, JsonValue>;
+  toolMeta?: Record<string, string>;
   transport?: "sse" | "stdio" | "http";
   headers?: Record<string, string>;
+  metaData?: Record<string, string>;
```

Comment:
As for `metaData` I can see it has been threaded through all the calls, but there are no tests. Can you add cli tests for the calls that pass something in `metaData`?